tox.ini

```diff
@@ -3,20 +3,20 @@ envlist = py27, py27-tornado23, py27-tornado31, docs, flake8
 
 [testenv]
 sitepackages = true
-commands = nosetests -v --with-xunit --xunit-file=xunit-{envname}.xml --with-coverage --cover-package=mopidy
+commands = py.test --junit-xml=xunit-{envname}.xml --cov=mopidy
```

Comment:
Could this be changed to using pytest-xdist to distribute the test run to more processes?
http://tox.readthedocs.org/en/latest/example/pytest.html#using-multiple-cpus-for-test-runs